### ISSUE TITLE
After applying orientation "Clear all" button is not visible and data present besides to Pie chart is truncating.:A11y_.NET 9 template_Home_Orientation

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27421]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG1.3.4; #A11ySev2; #iOS; #SH_.NET9template_iOS_Jan2025; #.NET Core; #Device; #Orientation; #A11yWCAG2.1; #AINotInscope; #iphone;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Apply Orientation.
4. Observe that After applying orientation "Clear all" button is not visible and data present besides to Pie chart is truncating.

### Actual Result:
After applying orientation "Clear all" button is not visible and data present besides to Pie chart is truncating.

### Expected Result:
After applying orientation "Clear all" button should be clearly visible and data present besides to Pie chart should not be truncating.

### User Impact:
Users with dexterity impairments will encounter difficulties as they will not be able to access the "Clear All" button. Additionally, the data next to the pie chart, visible in landscape mode, is missing for these users.

### Attachment:

https://github.com/user-attachments/assets/16854512-3f28-4b33-b47a-6332ba0bb897

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:03-06-25; #A11yFixed;

Verified the issue in the latest environment, issue is fixed hence closing the bug.

https://github.com/user-attachments/assets/787bcd8a-bcc2-4276-8265-3d6c5ed20c66
